src/renovate.ts

```diff
@@ -41,6 +49,35 @@ class Renovate {
     }
   }
 
+  /**
+   * Fetch the host docker group of the GitHub Action runner.
+   *
+   * The Renovate container needs access to this group in order to have the
+   * required permissions on the Docker socket.
+   */
+  private getDockerGroupId(): string {
+    const groupFile = '/etc/group';
+    const groups = fs.readFileSync(groupFile, {
+      encoding: 'utf-8',
+    });
+
+    /**
+     * The group file has `groupname:group-password:GID:username-list` as
+     * structure and we're interested in the `GID` (the group ID).
+     *
+     * Source: https://www.thegeekdiary.com/etcgroup-file-explained/
+     */
+    const re = new RegExp(`^${this.dockerGroupName}:x:([1-9][0-9]*):`, 'm');
```

Comment:
use static regex and use named capture group

Reply:
honestly, I just took a function that [was part of your codebase](https://github.com/renovatebot/github-action/commit/3aa72249b9d8787c333da7dcd0c6f1a92170e869) and brought it back. But okay I can rework that!